pkg/services/searchV2/http.go

```diff
@@ -36,8 +37,11 @@ func (s *searchHTTPService) doQuery(c *models.ReqContext) response.Response {
 			"reason": searchReadinessCheckResp.Reason,
 		}).Inc()
 
-		bytes, err := (&data.Frame{
-			Name: "Loading",
+		bytes, err := (&backend.DataResponse{
```

Comment:
In another [55878](https://github.com/grafana/grafana/pull/55878/files#diff-af5ede915211fa712b88460acee9a832a6f95747c11f504991b53e4279c326e0R90) I update this so we just return the response, we do not need to call `MarshalJSON` directly

Reply:
nice - I also extracted it to a separate PR since it needs to be backported to 9.2.0